parking_permits_app/pricing/secondary_vehicle.py

```diff
@@ -0,0 +1,5 @@
+from parking_permits_app.constants import SECONDARY_VEHICLE_PRICE_INCREASE
+
+
+def apply_secondary_vehicle_price_increase(price=None):
+    return (price / 100) * SECONDARY_VEHICLE_PRICE_INCREASE
```

Comment:
IMHO it should be `return price +  (price / 100) * SECONDARY_VEHICLE_PRICE_INCREASE`